hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+
+# 2. Typical regulatory certification regions
+Silicon Labs has many products such as chips, SiP modules, and PCB modules. And the products support main sub-GHz bands(169MHz,434MHz,868MHz and 915MHz) and 2.4GHz frequenc band. These different products and frequency bands almost comply to regulations from any region in the world. There are many region regulations such as [CE](http://ec.europa.eu/growth/sectors/electrical-engineering/red-directive_fi) (Europe), [FCC](https://www.fcc.gov/)(USA), [ISED](https://www.canada.ca/en/innovation-science-economic-development.html)(Canada), [MIC](http://www.soumu.go.jp/english/)(Japan), [KC](http://eng.kcc.go.kr/user/ehpMain.do)(South Korea) and other regions. 
+
+And there are some test houses provide certification service for the regions as following:
+<div align="center">
+  <img src="files\HW-Comply-to-the-Regulations\Test-House.png">  
+</div> 
+
+For the certification activeties, you can directly contact the region authorities:
+<div align="center">
+  <img src="files\HW-Comply-to-the-Regulations\Authorities-Contacts.png">  
+</div>
+
+In the following introduction, it will mainly focus on EC and FCC regulations. For Other region regulatory certifications, please check [AN1048](https://www.silabs.com/documents/public/application-notes/an1048-regulatory-certifications.pdf) for more details.
+Though there are RF, EMC and Safety separate categories to be certified.  RF certification is mainly introduced here.
+
+# 3. Key region regulatories introduction
+In this chapter, it will introduces FCC and CE regulations for both sub-GHz and 2.4GHz frequency bands. 
+  
+  ## 3.1. FCC Regulation 
+
+   FCC radio certification includes the standards such as intentional radiators 47 CFR FCC Part 15C and Unintentional radiators 47 CFR FCC Part 15B. And it can certificates an end product or radio module. For certificating modules, it includes limit modular approval(LMA) and full modular approval(FMA).
+   - FMA
+     
+     FMA is defined as: a completely self-contained RF transmitter that is missing only an input signal and power source to make it functional. It can be used in any host, its compliance is ensured independent of the host board typically. A full modular needs to meet all 8 criteria sets below: 
+     - 1. Transmitter / RF portion must have its own metallic shield.
```

Comment:
This 8 item list includes both bullet and letter.  We should use one or the other.  Is it intended to include a numbered list?

Reply:
Corrected. Yes, it will use numbered list.